content/glossary/baseflow.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Baseflow
+---
+
+Portion of the stream flow that is not from precipitation and results from seepage of water from the ground.
+
+Baseflow projections on Cal-Adapt are produced using a land surface and hydrology model known as the [Variable Infiltration Capacity](/help/glossary/) (VIC) model.
```

Comment:
add hash to glossary link

Reply:
@mukhtyar to link to any term in the glossary use a hash version the glossary term's `metadata.title` and replace all spaces with "-" and convert it to lowercase. Then you can use the glossary path and hash term, e.g.

```
/help/glossary/#variable-infiltration-capacity-(vic)
```

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/69cb77de4d70c5bd1c8af193d3592b86a0a44ea7